Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:11-jre as with_java
+
+FROM golang:1.20 
```

Comment:
does it matter when this version is not consistent with 1.19 in go.mod?

Reply:
both works on `go run` 

I noticed differences in the size of the docker image.  `cc3a4c313856` is golang 1.20.  `dcf1b1bad329` is golang 1.19

```
REPOSITORY         TAG           IMAGE ID       CREATED          SIZE
sprinkler_ws       latest        cc3a4c313856   2 days ago       915MB

REPOSITORY         TAG           IMAGE ID       CREATED          SIZE
sprinkler_ws       latest        dcf1b1bad329   16 seconds ago   1.13GB
```

Seems 1.20 is smaller, any objections to switching to 1.20 in the `go.mod`?

Reply:
@schowsf no objection my end

Reply:
^^ I mean mix of 1.19 and 1.20 between `Dockerfile` and `go.mod`, or consistently 1.19 or 1.20.  all combinations worked.

Reply:
go for it

Reply:
I'm for 1.20 for both.

Reply:
let's make everything consistent and latest stable version

Reply:
okay.  1.20 commit pushed.